### DISCUSSION TITLE
Manual bed leveling menu Z Position va;ue

### DISCUSSION THREAD

**Discussion by MicroSur** [Jyers/Marlin#1413] [Q&A] [answered]
Manual Leveling menu has Z Position item, maximum 9.99. Can I raise this height more in the firmware source? Where?

**Comment by Jyers** [comment, marked as answer]
Why would you ever need a Z position over 10?

**Reply by MicroSur** [reply]
I found this setting in creality_dwin.cpp. 
Wanted to try the idea of bed leveling using a strain sensor )

**Reply by Jyers** [reply]
Interesting, well glad you found it 😄
